subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/buildcache/TaskOutputCachingSwiftPerformanceTest.groovy

```diff
@@ -19,32 +19,24 @@ package org.gradle.performance.regression.buildcache
 import org.gradle.initialization.ParallelismBuildOptions
 import org.gradle.util.Requires
 import org.gradle.util.TestPrecondition
-import spock.lang.Unroll
 
-@Requires(TestPrecondition.LINUX)
+@Requires(TestPrecondition.NOT_WINDOWS)
```

Comment:
I think it's better to have an exhaustive list of OS where we do want to run these things. WDYT?

Reply:
Yes, I think the whole `@Requires` annotations should go away.

Reply:
I removed the `@Requires` annotations.